.pre-commit-config.yaml

```diff
@@ -1,6 +1,6 @@
 repos:
   - repo: .
-    rev: 1.2.0
+    rev: 1.3.0
```

Comment:
I've never actually used git tags before.  If this is merged do I tag the commit that was merged into master?